### PR TITLE
Add sbt release job

### PR DESCRIPTION
**Pull request by MancunianSam**
This will run sbt release.

The GPG secrets are for deploying this to maven.

### CODE REVIEW

File: .github/workflows/sbt_release.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    steps:
+      - uses: actions/checkout@v3
+      - name: Import GPG key
+        id: import_gpg
+        uses: crazy-max/ghaction-import-gpg@v4
+        with:
+          gpg_private_key: ${{ secrets.GPG_PRIVATE_KEY }}
+          passphrase: ${{ secrets.GPG_PASSPHRASE }}
+      - run: |
+          git config --global user.email digitalpreservation@nationalarchives.gov.uk
```
Comment (TomJKing) [inline .github/workflows/sbt_release.yml:39]: Who owns this email address? Should we be using something else?

**Review by TomJKing — COMMENTED** [review]
Looks good. One minor comment, feel free to ignore, as think I'm worrying about nothing

File: .github/workflows/sbt_release.yml (hunk 1 above)
Comment (MancunianSam) [reply .github/workflows/sbt_release.yml:39]: It's what we've been using in the Jenkins git config. I don't think anything actually sends emails to it anyway, it just shows up on the published packages.
I think someone does maintain that mailbox though.

Comment (TomJKing) [reply .github/workflows/sbt_release.yml:39]: OK. Might be worth finding out, just to be sure about that email address.

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks.

**Pull request merged**